docs/cli/cli.md

```diff
@@ -41,6 +41,11 @@ Consider the following before setting up the integration using the Pulse CLI:
 
 Download the latest version of the Pulse CLI for your operating system and make sure that you're able to run the binary.
 
+!!! tip
+    **If you're using GitHub** we recommend that you use our [Pulse GitHub Action](https://github.com/codacy/pulse-action) as a more convenient way to report events directly from your GitHub Actions workflows.
```

Comment:
Well, if you're using GitHub our first recommendation is to use the integration. :)

```suggestion
    **If you're using GitHub** and our [integration](link to integration page) does not fit your requirements, we recommend that you use our [Pulse GitHub Action](https://github.com/codacy/pulse-action) as a more convenient way to report events directly from your GitHub Actions workflows.
```

Reply:
:facepalm: :sweat_smile: 

I completely forgot about that! :laughing: 

What would be the ideal scenario or use case for using the Pulse GitHub Action then? :thinking: I know it was created before we had the GitHub integration. Maybe it's not so relevant anymore? :thinking:

Reply:
OK, I believe it still makes sense to mention the alternative - it's up to the users to decide which solution works best for them.

I adjusted the wording based on your suggestion @afsmeira, please check if you agree:

https://deploy-preview-96--docs-pulse.netlify.app/cli/cli/#1-installing-the-pulse-cli